lib/valkyrie/specs/shared_specs/change_set_persister.rb

```diff
@@ -2,25 +2,25 @@
 RSpec.shared_examples 'a Valkyrie::ChangeSetPersister' do |*_flags|
   before do
     raise 'adapter must be set with `let(:change_set_persister)`' unless defined? change_set_persister
-    class CustomResource < Valkyrie::Resource
+    class Valkyrie::Specs::CustomResource < Valkyrie::Resource
       include Valkyrie::Resource::AccessControls
       attribute :title
       attribute :member_ids
       attribute :nested_resource
     end
-    class CustomChangeSet < Valkyrie::ChangeSet
+    class Valkyrie::Specs::CustomChangeSet < Valkyrie::ChangeSet
       self.fields = [:title]
     end
   end
   after do
-    Object.send(:remove_const, :CustomResource)
-    Object.send(:remove_const, :CustomChangeSet)
+    Valkyrie::Specs.send(:remove_const, :Valkyrie::Specs::CustomResource)
+    Valkyrie::Specs.send(:remove_const, :Valkyrie::Specs::CustomChangeSet)
```

Comment:
Is there a reason these two `remove_const` are passed the full classpath instead of just the relative `:CustomResource`?  The other shared specs in this PR still use the relative form.

Reply:
Because I forgot to remove it. This would actually break, thanks for catching it.